### PR TITLE
Add alpha2 changelog

### PR DESCRIPTION
**Pull request by 1st1**

### CODE REVIEW

**Comment by 1st1** [comment]
Beware: my VSCode doesn't have a spell checker, so please look for typos. I also didn't have time to re-read everything and do a grammar check. Suggestions/fixes are welcome.

File: docs/changelog/1_0_a2.rst [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+EdgeDB 1.0 alpha 2.
+
+
+JavaScript Driver
+=================
+
+EdgeDB has a new high-performance native
+`EdgeDB driver <https://github.com/edgedb/edgedb-js>`_ for NodeJS 10+.
+
+It is written in strict TypeScript, thouroughly tested, and has a
```
Comment (RobertoPrevato) [inline docs/changelog/1_0_a2.rst]: There is a typo here: it should be _thoroughly tested_.

File: docs/changelog/1_0_a2.rst [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+:eql:type:`std::decimal`.
+
+
+New "cal" Module
+----------------
+
+Non-timezone aware date/time types and functions are moved into the new
+``cal`` module.  This new separation promotes :eql:type:`std::datetime`
+to be the default safe choice for most use cases.  The types in the ``cal::``
+module are useful to implement calendars, alarms, reminders, etc, but have
```
Comment (RobertoPrevato) [inline docs/changelog/1_0_a2.rst]: "etc" should always be followed by a trailing period. Hence, "etc., but ..."
([1.](https://english.stackexchange.com/questions/188927/can-we-use-the-abbreviation-etc-without-a-period-full-stop) [2.](https://www.editage.com/insights/research-paper-comma-after-etc-at-the-end-of-a-sentence))

File: docs/changelog/1_0_a2.rst [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+listed below:
+
+.. list-table::
+    :class: funcoptable
+
+    * - :eql:type:`std::datetime`
+      - A timezone-aware date/time type.
+
+    * - :eql:type:`std::duration`
+      - An absolute time interval. Can be unabmigously used with
```
Comment (RobertoPrevato) [inline docs/changelog/1_0_a2.rst]: Typo: it should be `unambiguously`

File: docs/changelog/1_0_a2.rst [hunk 4]
```diff
@@ -0,0 +1,475 @@
...
+----------------------------
+
+* Fix backward links in aliases (:eql:gh:`#990`).
+* Fix covarinat types support (:eql:gh:`#709`).
+
+
+DDL / SDL / Schema
+==================
+
+DDL and SDL layers are havily refactored in alpha 2. A lot of issues were
```
Comment (RobertoPrevato) [inline docs/changelog/1_0_a2.rst]: Typo: _"havily"_ instead of _"heavily"_

File: docs/changelog/1_0_a2.rst [hunk 5]
```diff
@@ -0,0 +1,475 @@
...
+Other Fixes and Enhancements
+----------------------------
+
+* Add an explicit database instance compatibility check (:eql:gh:`251517c0`).
+* Initial support for using a remote Postgres cluster as a backend
+  (:eql:gh:`b0db89b2`).
+* Protocol: prohibit tuples as query arguments (:eql:gh:`#745`).
+* Protocol: differentiate SASL message types (:eql:gh:`d52885c8`).
+* Protocol: Add "Terminate" message for graceful shutdown (:eql:gh:`d699352a`).
+* Protocol: use 32-bit lentgh-prefixed strings everywhere.
```
Comment (RobertoPrevato) [inline docs/changelog/1_0_a2.rst]: Typo: should be _length-prefixed_

File: docs/changelog/1_0_a2.rst [hunk 6]
```diff
@@ -0,0 +1,475 @@
...
+* Rename ``schema::volatility_t`` to ``schema::Volatility``
+  (:eql:gh:`16e263cc`).
+
+
+Dump / Restore
+==============
+
+The new :ref:`edgedb dump <ref_cli_edgedb_dump>` and
+:ref:`edgedb restore <ref_cli_edgedb_restore>` commands can be used to
+safely dump and restore EdgeDB databases.
```
Comment (tailhook) [inline docs/changelog/1_0_a2.rst]: Mention that it should be used for database upgrades?

File: docs/changelog/1_0_a2.rst [hunk 7]
```diff
@@ -0,0 +1,475 @@
...
+    SELECT User.<profile[IS Profile].settings;
+
+(See :eql:gh:`#969` for details.)
+
+Other Fixes and Enhancements
+----------------------------
+
+* Update the semantics of line continuation (trailing ``\``) in strings
+  (:eql:gh:`#921`).
+* Remove the ``.>`` syntax for forward links navigation (:eql:gh:`#982`).
```
Comment (tailhook) [inline docs/changelog/1_0_a2.rst]: Mention:  `.` has the same functionality

File: docs/changelog/1_0_a2.rst [hunk 8]
```diff
@@ -0,0 +1,475 @@
...
+* Protocol: Add "Terminate" message for graceful shutdown (:eql:gh:`d699352a`).
+* Protocol: use 32-bit lentgh-prefixed strings everywhere.
+* Drop reliance on a custom PostgreSQL C extension.
+
+
+Misc
+====
+
+* Command-line tools now use ``-h`` for help; ``-H`` for hostname
+  (:eql:gh:`#1039`).
```
Comment (tailhook) [inline docs/changelog/1_0_a2.rst]: Don't see `create role` -> `create-role` CLI change documented.

**Comment by vpetrovykh** [comment]
Nothing really jumped out at me beyond the already mentioned typos. I'll see if anything strikes me as I compare the changelog to the actual docs.

File: docs/changelog/1_0_a2.rst [hunk 9]
```diff
@@ -0,0 +1,475 @@
...
+Standard Library
+================
+
+std::bigint
+-----------
+
+The new :eql:type:`std::bigint` scalar type is an arbitrary precision integral
+type.  The motivation for the new type is that many platforms lack a true
+multi-precision decimal type, but implement an arbitrary-precision integer
+type (JavaScript is a prominent example).  The decimal literals syntax
```
Comment (elprans) [inline docs/changelog/1_0_a2.rst]: "The syntax for decimal literals" reads a bit better imo

**Comment by 1st1** [comment]
Thanks for reviews!

**Comment by 1st1** [comment]
Merged by hand.

**Pull request closed without merging**